### PR TITLE
chore(protocol): update VDP protocol

### PR DESCRIPTION
**Pull request by praharshjain**
Because

- VDP protocol is outdated

This commit

- Updates VDP protocol with -

1. Input and output format for all tasks
2. TextEmbeddings task
3. ImageToImage task

### CODE REVIEW

File: protocol/vdp_protocol.yaml [hunk 1]
```diff
@@ -5,273 +5,570 @@ title: VDP Protocol
...
+  - ocr
+- required:
+  - instance_segmentation
+- required:
+  - semantic_segmentation
+- required:
+  - text_to_image
+- required:
+  - text_generation
+- required:
```
Comment (donch1989) [inline protocol/vdp_protocol.yaml:25]: The new tasks is missing here

File: protocol/vdp_protocol.yaml [hunk 2]
```diff
@@ -5,273 +5,570 @@ title: VDP Protocol
...
-        description: "The confidence score of the predicted category of the input"
-        type: number
-  Detection:
-    type: object
-    additionalProperties: false
-    required:
-      - objects
+    oneOf:
+    - required:
+      - input
```
Comment (donch1989) [inline protocol/vdp_protocol.yaml:61]: I suggest we use plural `inputs` and `outputs`

File: protocol/vdp_protocol.yaml (hunk 1 above)
Comment (praharshjain) [reply protocol/vdp_protocol.yaml:27]: updated

File: protocol/vdp_protocol.yaml (hunk 2 above)
Comment (praharshjain) [reply protocol/vdp_protocol.yaml]: changed

**Comment by praharshjain** [comment]
This is not needed now.

**Pull request closed without merging**